### PR TITLE
Removed content, Fixed javadocs

### PR DESCRIPTION
**Pull request by politeRaccoon**

### CODE REVIEW

File: src/main/java/org/mitre/caasd/commons/YyyyMmDd.java [hunk 1]
```diff
@@ -136,4 +141,31 @@ public static YyyyMmDd newestOf(Collection<YyyyMmDd> dates) {
         checkNotNull(dates);
         return dates.stream().max(Comparator.naturalOrder()).orElse(null);
     }
+
+
+    private static final String DATE_FORMAT_ERROR = "Date not specified as YYYY-MM-DD format";
+
+    /** Throw an IllegalArgumentException if the input String is not in the YYYY-MM-DD format. */
+    public static void verifyYearMonthDayFormat(String date) {
```
Comment (stellarsunset) [inline src/main/java/org/mitre/caasd/commons/YyyyMmDd.java:149]: Can we not do this with a `DateTimeFormatter`?

File: src/main/java/org/mitre/caasd/commons/fileutil/FileUtils.java [hunk 2]
```diff
@@ -50,9 +50,6 @@
 
 public class FileUtils {
 
-    private FileUtils() {
```
Comment (stellarsunset) [inline src/main/java/org/mitre/caasd/commons/fileutil/FileUtils.java:53]: This is a static utility class - standard practice is the keep the constructor and have it throw `IllegalStateException` (at least according to most linters).

**Review by stellarsunset — COMMENTED** [review]
Rest in peace Michigan cities 🙏

File: src/main/java/org/mitre/caasd/commons/fileutil/FileUtils.java (hunk 2 above)
Comment (politeRaccoon) [reply src/main/java/org/mitre/caasd/commons/fileutil/FileUtils.java:53]: I've flip flopped on this.  My current view is it's slightly better to keep the class file cleaner.  I no longer think preventing someone from calling ` new FileUtils()` is a relevant critique.  That said.  I do not care much at all.  seems like argueing about the least important thing ever

File: src/main/java/org/mitre/caasd/commons/YyyyMmDd.java (hunk 1 above)
Comment (politeRaccoon) [reply src/main/java/org/mitre/caasd/commons/YyyyMmDd.java:149]: You'd think so right.  I am not aware of how to cleanly do this with a ISO_LOCAL_DATE formatter.  I could be missing something obvious.  But manually implementing this wasn't my first choice.

Comment (politeRaccoon) [reply src/main/java/org/mitre/caasd/commons/YyyyMmDd.java:149]: You can..

```
        try {
            ISO_LOCAL_DATE.parse(date);
        } catch(DateTimeParseException dtpe) {
            throw new IllegalArgumentException("ISO_LOCAL_DATE cannot parse: " + date);
        }
```

Comment (politeRaccoon) [reply src/main/java/org/mitre/caasd/commons/YyyyMmDd.java:151]: I rewrote the commit to add the above code

**Comment by politeRaccoon** [comment]
>

File: src/main/java/org/mitre/caasd/commons/fileutil/FileUtils.java (hunk 2 above)
Comment (stellarsunset) [reply src/main/java/org/mitre/caasd/commons/fileutil/FileUtils.java:53]: Yeah I don't really care either - just noting the convention or whatever you want to call it.

**Pull request merged**